SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/handshake-org/hsd/releases).
+We recommend using the [most recently released version](https://github.com/handshake-org/hsd/releases/latest).
+
+## Reporting a Vulnerability
+
+To report security issues send an email to chjjeffrey@gmail.com (not for support).
+
+The following keys may be used to communicate sensitive information to developers:
+
+| Name | Fingerprint | Full Key |
+|------|-------------|--------|
+| Christopher Jeffrey (@chjj) | B4B1F62DBAC084E333F3A04A8962AB9DE6666BBD | https://keybase.io/chjj |
+
```

Comment:
I think @pinheadmz should be on this list as well as a fallback in case there is no response from @chjj. This would help to diffuse responsibility so that chjj can take time off without worrying about things if he decides to.

Reply:
added and rebased.